ssat.sh

```diff
@@ -217,6 +217,10 @@ do
 	if [[ "$PROGRESS" -eq "1" ]]; then
 		printf "[Starting] $CASENAME\n"
 	fi
+
+	printf "${Green}==================================================${NC}\n"
+	printf "    Test Group ${Green}$CASENAME${NC} Starts.\n"
+
```

Comment:
If you need this, use ```-p```.
If you need this for each "casename" only, let's define ```-p={$n}```, where n is a number of "log level".